org.skytemple.SkyTemple.yml

```diff
@@ -0,0 +1,144 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/bin/run.sh
+finish-args:
+  - "--share=network"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--filesystem=host"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+cleanup:  # todo
+  - /bin/pcap-config
+  - /include
+  - /lib/libpcap.a
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: armips
+    buildsystem: simple
+    build-commands:
+      - mkdir -p build
+      - patch Core/ExpressionFunctions.cpp < armips.patch
```

Comment:
this is not how you apply patches.

"type": "patch" in the source will take care of it.